packages/fxa-settings/src/components/Settings/BentoMenu/index.tsx

```diff
@@ -33,6 +35,13 @@ export const BentoMenu = () => {
   const { env } = useConfig();
   const ftlMsgResolver = useFtlMsgResolver();
 
+  useEffect(() => {
+    if (isRevealed && !hasLoggedViewEvent) {
+      GleanMetrics.accountPref.bentoView();
+      setHasLoggedViewEvent(true);
```

Comment:
Is the intent that we only want to ever log 1 of these view events even if the user clicks on it, closes, and re-opens?

Reply:
That was my intent, but it might not be necessary.